pkg/guest/guest.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/labring/sealos/fork/golang/expansion"
 	"github.com/labring/sealos/pkg/constants"
 	"github.com/labring/sealos/pkg/env"
-	"github.com/labring/sealos/pkg/runtime"
+	runtime "github.com/labring/sealos/pkg/runtime/kubernetes"
```

Comment:
Can you also change it here, the guest should not pay attention to the real-time details of the runtime